docs/api/graphql_queries.md

```diff
@@ -181,6 +181,72 @@ Response:
 }
 ```
 
+### Get Matrix Field Type
+
+To get a Matrix Field Type, use `product` field.
```

Comment:
What `product` field ? In any case, whatever the field, an example content type & field is needed, it isn't specific to Matrix.

Also, in the content type described below, `product` isn't the field, it is the content type. The field is `features`.